### PR TITLE
Added back mpMetrics 1.1 fat tck project

### LINKED ISSUE #2356: Microprofile Metrcs 1.1 TCK
Run Microprofile Metrics 1.1 TCK against Liberty mpMetrics-1.1 feature

### PR DESCRIPTION
**Pull request by raymondlam**
Disabled timing related tests, see #2493 to prevent them from failing.

Fixes #2356

### CODE REVIEW

**Comment by raymondlam** [comment]
#build

File: dev/com.ibm.ws.microprofile.metrics.1.1_fat_tck/publish/servers/MetricsTCKServer/server.xml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<server>
+    <featureManager>
+    	<feature>mpMetrics-1.1</feature>
+    	<feature>mpConfig-1.2</feature>
```
Comment (fmhwong) [inline dev/com.ibm.ws.microprofile.metrics.1.1_fat_tck/publish/servers/MetricsTCKServer/server.xml]: Do we still need mpConfig-1.2 as we already have com.ibm.websphere.appserver.mpConfig-1.2 in the feature file.

File: dev/com.ibm.ws.microprofile.metrics.1.1_fat_tck/publish/tckRunner/tck/pom.xml [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+		<dependency>
+			<groupId>javax.enterprise</groupId>
+			<artifactId>cdi-api</artifactId>
+			<version>1.2</version>
+			<scope>provided</scope>
+		</dependency>
+        <dependency>
+            <groupId>com.ibm.ws.org.jboss.arquillian.container</groupId>
+            <artifactId>arquillian-wlp-managed-8.5</artifactId>
+            <version>1.0.0.Final-SNAPSHOT</version>
```
Comment (fmhwong) [inline dev/com.ibm.ws.microprofile.metrics.1.1_fat_tck/publish/tckRunner/tck/pom.xml]: Do we need to change to the specific version they have in the artifactory?

File: dev/com.ibm.ws.microprofile.metrics_fat_tck/publish/tckRunner/tck/pom.xml [hunk 3]
```diff
@@ -124,6 +124,9 @@
                     </dependenciesToScan>
                     <excludes>
                        <exclude>**/TimerTest.java</exclude>
+                       <exclude>**/MpMetricTest.java</exclude>
```
Comment (fmhwong) [inline dev/com.ibm.ws.microprofile.metrics_fat_tck/publish/tckRunner/tck/pom.xml]: This is the 1.0 pom.xml file.  Is this change already contained in PR #2493?

File: dev/com.ibm.ws.microprofile.metrics.1.1_fat_tck/publish/tckRunner/tck/pom.xml [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+					<systemPropertyVariables>
+						<wlp>${wlp}</wlp>
+						<tck_server>${tck_server}</tck_server>
+                        <tck_port>${tck_port}</tck_port>
+					</systemPropertyVariables>
+                    <dependenciesToScan>
+                        <dependency>org.eclipse.microprofile.metrics:microprofile-metrics-rest-tck</dependency>
+                        <dependency>org.eclipse.microprofile.metrics:microprofile-metrics-api-tck</dependency>
+                    </dependenciesToScan>
+               </configuration>
```
Comment (fmhwong) [inline dev/com.ibm.ws.microprofile.metrics.1.1_fat_tck/publish/tckRunner/tck/pom.xml]: Are we missing the excludes here?

**Comment by raymondlam** [comment]
forgot to force push changes.
Removed mpConfig from server.xml

File: dev/com.ibm.ws.microprofile.metrics.1.1_fat_tck/publish/tckRunner/tck/pom.xml (hunk 2 above)
Comment (raymondlam) [reply dev/com.ibm.ws.microprofile.metrics.1.1_fat_tck/publish/tckRunner/tck/pom.xml]: I double checked, they only switched to a specific version for stabilization (for repeatability). There isn't an issue opened to switch it in integration yet.

**Comment by raymondlam** [comment]
#build

**Pull request merged**